### PR TITLE
Dev feature upload status query updates

### PR DESCRIPTION
**Pull request by mtammineni**

### CODE REVIEW

File: pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/loaders/UploadStatusLoader.kt [hunk 1]
```diff
@@ -62,53 +63,66 @@ class UploadStatusLoader: CosmosLoader() {
...
-                //"date" -> "_ts" // "group  by _ts" is already added by default above
+                "date" -> "dexIngestDateTime" // "group  by _ts" is already added by default above
                 "fileName" -> "content.filename"
-                "dataStreamId" -> "destinationId"
-                "dataStreamRoute" -> "eventType"
-                "stageName" -> "stageName"
-                "jurisdiction" -> "content.metadata.meta_ext_source"
+                "dataStreamId" -> "dataStreamId"
+                "dataStreamRoute" -> "dataStreamRoute"
+                "stageName" -> "stageInfo.service"
```
Comment (mkrystof) [inline pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/loaders/UploadStatusLoader.kt:88]: Please rename "stageName" to "service" -> "stageInfo.service" and add "action" -> "stageInfo.action".

File: pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/models/query/PageSummary.kt [hunk 2]
```diff
@@ -24,5 +24,11 @@ data class PageSummary(
     var pageSize: Int = 0,
 
     @GraphQLDescription("Total number of items that can be provided")
-    var totalItems: Int = 0
+    var totalItems: Int = 0,
+
+    @GraphQLDescription("List of the senderIds")
+    var senderIds: List<String> = listOf(),
+
+    @GraphQLDescription("List of the jurisdictions")
```
Comment (mkrystof) [inline pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/models/query/PageSummary.kt:32]: Change the description to be more informative.  Something like, "List of all the jurisdictions in the entire dataset matching the filter criteria, not just this page."

File: pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/models/query/PageSummary.kt [hunk 3]
```diff
@@ -24,5 +24,11 @@ data class PageSummary(
     var pageSize: Int = 0,
 
     @GraphQLDescription("Total number of items that can be provided")
-    var totalItems: Int = 0
+    var totalItems: Int = 0,
+
+    @GraphQLDescription("List of the senderIds")
```
Comment (mkrystof) [inline pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/models/query/PageSummary.kt:29]: Change the description to be more informative.  Something like, "List of all the senderIds in the entire dataset matching the filter criteria, not just this page."

File: pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/models/query/UploadStatus.kt [hunk 4]
```diff
@@ -67,7 +67,7 @@ class UploadStatus {
             // Convert the reports to their schema objects
             val reportsWithSchemaPairs = mutableListOf<Pair<SchemaDefinition, ReportDao>>()
             reports.forEach { report ->
-                if (report.contentType != "json")
+                if (report.contentType != "application/json")
```
Comment (mkrystof) [inline pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/models/query/UploadStatus.kt:70]: Make is so contentType can be "json" or "application/json".

File: pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/models/reports/MetadataVerifyContent.kt [hunk 5]
```diff
@@ -6,10 +6,10 @@ import com.expediagroup.graphql.generator.annotations.GraphQLDescription
 class MetadataVerifyContent : SchemaDefinition {
 
     @GraphQLDescription("Schema name for the report content")
-    override val schemaName: String = "dex-metadata-verify"
+    override val schemaName: String = "upload-status"
```
Comment (mkrystof) [inline pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/models/reports/MetadataVerifyContent.kt:9]: Why was this changed?  This should still be the metadata verify one.

File: pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/models/reports/stagereports/UploadMetadataVerifyStage.kt [hunk 6]
```diff
@@ -16,6 +16,6 @@ class UploadMetadataVerifyStage: SchemaDefinition(priority = -1) {
     var issues: List<String>? = null
 
     companion object {
-        val schemaDefinition = SchemaDefinition(schemaName = "dex-metadata-verify", schemaVersion = "0.0.1")
+        val schemaDefinition = SchemaDefinition(schemaName = "upload-status", schemaVersion = "1.0.0")
```
Comment (mkrystof) [inline pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/models/reports/stagereports/UploadMetadataVerifyStage.kt:19]: I don't understand why this was changed from metadata verify to upload status.  Would you please explain?

**Review by mkrystof — CHANGES_REQUESTED** [review]

File: pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/loaders/UploadStatusLoader.kt (hunk 1 above)
Comment (mtammineni) [reply pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/loaders/UploadStatusLoader.kt]: Updated according to the suggestions

File: pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/models/query/PageSummary.kt (hunk 2 above)
Comment (mtammineni) [reply pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/models/query/PageSummary.kt]: Updated according to the suggestions

File: pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/models/query/PageSummary.kt (hunk 3 above)
Comment (mtammineni) [reply pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/models/query/PageSummary.kt]: Updated according to the suggestions

File: pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/models/query/UploadStatus.kt (hunk 4 above)
Comment (mtammineni) [reply pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/models/query/UploadStatus.kt]: Updated according to the suggestions

File: pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/models/reports/MetadataVerifyContent.kt (hunk 5 above)
Comment (mtammineni) [reply pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/models/reports/MetadataVerifyContent.kt]: This change is reverted for the Metadata classes. 
However, upon closer inspection, noticed that the default SchemaDefinition values should be set to the following:
_companion object {
        val schemaDefinition = SchemaDefinition(schemaName = "upload-status", schemaVersion = "1.0.0")
    }_

The content_schema_name and content_schema_version for the upload is updated to "upload-status", "1.0.0" respectively in the reports.py scripts

File: pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/models/reports/stagereports/UploadMetadataVerifyStage.kt (hunk 6 above)
Comment (mtammineni) [reply pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/models/reports/stagereports/UploadMetadataVerifyStage.kt]: This change is reverted for the Metadata classes. 
However, upon closer inspection, noticed that the default SchemaDefinition values should be set to the following:
_companion object {
        val schemaDefinition = SchemaDefinition(schemaName = "upload-status", schemaVersion = "1.0.0")
    }_

The content_schema_name and content_schema_version for the upload is updated to "upload-status", "1.0.0" respectively in the reports.py scripts

**Review by mkrystof — APPROVED** [review]
LGTM

**Comment by mtammineni** [comment]
GetUploadsStatus API updates for sort and filter.
Update pstatus-graph-ktor to use new field for date and time, dexIngestDateTime.
